spec/spec_helper.rb

```diff
@@ -14,13 +14,6 @@
 require 'vcr'
 require 'cgi'
 
-# FIXME: This bypasses the lookup of Javascript dependencies in tests until they can be found and uncoupled
-class << ActionController::Base.helpers
-  def image_path(path, options = {})
-    path
-  end
-end
-
```

Comment:
..Tested, seems it works now :) (I only get 2 postgres-version related failures (no `waiting` in `pg_stat_activity`).)